src/components/popupwithconfirmation.js

```diff
@@ -0,0 +1,30 @@
+import Popup from "./Popup.js";
+
```

Comment:
`popupwithconfirmation.js` should be in **camelCase**, and it should start with a capital letter, like the class: `PopupWithConfirmation.js`